src/Maestro/DependencyUpdateErrorProcessor/.config/settings.Staging.json

```diff
@@ -0,0 +1,8 @@
+{
+  "KeyVaultUri": "https://maestroint.vault.azure.net/",
```

Comment:
Should this really be running for staging?

Reply:
don't we want this feature to run in staging? Yes :)

Reply:
We do? What's the user experience going to be like? where are the issues going to be opened at?

Reply:
I want some baketime for this in staging. So I thought I will create errors and see if those creates issues/

Reply:
makes sense